src/applications/vaos/services/location/index.js

```diff
@@ -369,10 +370,28 @@ export async function fetchParentLocations({ siteIds }) {
  *
  * @export
  * @param {Object} params
- * @param {Array<Location>} locations The locations to find CC support at
+ * @param {Array<Location>} params.locations The locations to find CC support at
+ * @param {boolean} params.useV2 Use the V2 scheduling configurations endpoint
+ *   to get the CC supported locations
  * @returns {Array<Location>} A list of locations that support CC requests
  */
-export async function fetchCommunityCareSupportedSites({ locations }) {
+export async function fetchCommunityCareSupportedSites({
+  locations,
+  useV2 = false,
+}) {
+  if (useV2) {
+    const facilityConfigs = await getSchedulingConfigurations(
+      locations.map(location => location.id),
+      true,
+    );
+
+    return locations.filter(location =>
```

Comment:
Assuming that since we're passing the `ccEnabled` flag, all the locations we get back should be cc enabled, so we just have to filter for those ones in our original list.